README.md

```diff
@@ -26,121 +26,8 @@ For getting started after cloning your own copy to your local machine, you shoul
-            arch: wasm32
-            os: ubuntu-20.04
-
-    runs-on: ${{ matrix.os }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          submodules: true
-      - name: 🔗 GDExtension Build
-        uses: godotengine/godot-cpp-template/.github/actions/build@main
-        with:
-          platform: ${{ matrix.platform }}
-          arch: ${{ matrix.arch }}
-          float-precision: single
-          build-target-type: template_release
-      - name: 🔗 GDExtension Build
-        uses: ./.github/actions/build
-        with:
-          platform: ${{ matrix.platform }}
-          arch: ${{ matrix.arch }}
-          float-precision: ${{ matrix.float-precision }}
-          build-target-type: template_debug
-      - name: Mac Sign
-        if: ${{ matrix.platform == 'macos' && env.APPLE_CERT_BASE64 }}
-        env:
-          APPLE_CERT_BASE64: ${{ secrets.APPLE_CERT_BASE64 }}
-        uses: godotengine/godot-cpp-template/.github/actions/sign@main
-        with:
-          FRAMEWORK_PATH: bin/macos/macos.framework
-          APPLE_CERT_BASE64: ${{ secrets.APPLE_CERT_BASE64 }}
-          APPLE_CERT_PASSWORD: ${{ secrets.APPLE_CERT_PASSWORD }}
-          APPLE_DEV_PASSWORD: ${{ secrets.APPLE_DEV_PASSWORD }}
-          APPLE_DEV_ID: ${{ secrets.APPLE_DEV_ID }}
-          APPLE_DEV_TEAM_ID: ${{ secrets.APPLE_DEV_TEAM_ID }}
-          APPLE_DEV_APP_ID: ${{ secrets.APPLE_DEV_APP_ID }}
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: GDExtension-${{ matrix.platform }}-${{ matrix.arch }}
-          path: |
-            ${{ github.workspace }}/bin/**
-  merge:
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Merge Artifacts
-        uses: actions/upload-artifact/merge@v4
-        with:
-          name: GDExtension-all
-          pattern: GDExtension-*
-          delete-merged: true
-```
-
-The above example is a lengthy one, so we will go through it action by action to see what is going on.
-
-In the `Checkout` step, we checkout the code.
-In the `🔗 GDExtension Build` step, we are using the reusable action:
-```yml
-uses: godotengine/godot-cpp-template/.github/actions/build@main
-with:
-  platform: ${{ matrix.platform }}
-  arch: ${{ matrix.arch }}
-  float-precision: single
-  build-target-type: template_release
-```
-with the parameters from the matrix.
-
-As a result of this step, the binaries will be built in the `bin` folder (as specified in the SConstruct file). After all builds are completed, all individual builds will be merged into one common GDExtension-all zip that you can download.
+This repository comes with a GitHub action that builds the GDExtension for cross-platform use. It triggers automatically for each pushed change. You can find and edit it in [builds.yml](.github/workflows/builds.yml).
+After a workflow run is complete, you can find a `GDExtension-all` zip on the `Actions` tab on GitHub.
```

Comment:
Isn't the final artifact named `godot-cpp-template` as it is called in the upload-artifact action call? This reminds me that we still have this remnant in 
`.github/workflows/builds.yml`:
```yaml
env:
  LIBNAME: example
```

Reply:
You're right! I just kinda assumed it would be the same as in the godot-cpp example from the docs. 

As for the `LIBNAME` thing, I was about to say it should be a separate PR, but it does appear the name is not used anywhere else in the project. I just removed it; builds will probably just fail if it was used anywhere important.